WordPress/src/main/java/org/wordpress/android/GCMIntentService.java

```diff
@@ -115,6 +116,7 @@ protected void handleDefaultPush(Context context, Bundle extras) {
         }
 
         String iconUrl = extras.getString("icon");
+        String noteType = StringUtils.notNullStr(extras.getString("type"));
```

Comment:
Don't think that'd work, it is looping through different bundles that might have different types.